### PR TITLE
Added modal style and functionality

### PR DESCRIPTION
**Pull request by LordLowdar**
This added some base style to the modal we dont need to keep it. I also made it possible for modal to trigger from the login link. It is also successfully requesting the login and verifying them.

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -45,4 +45,4 @@ li {
 
 .underline {
     text-decoration: underline;
-}
+}
```
Comment (Myuze) [inline public/css/main.css:48]:
```suggestion
}

```

File: public/css/modal.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+.modalContent {
+ font-family: monospace;
+
+}  
+
+#closeBtn {
+  display: flex;
+  justify-content: right;
+}
```
Comment (Myuze) [inline public/css/modal.css:24]:
```suggestion
}

```

File: views/layouts/main.handlebars [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {{! This will be the same doctype, and heading for every rendered document}}
 <!DOCTYPE html>
 <html>
-<head>
+<>
```
Comment (Myuze) [inline views/layouts/main.handlebars:4]: Typo? Why are these needed?

File: views/register.handlebars [hunk 4]
```diff
@@ -1 +1,34 @@
...
+
+    <p>By submitting information you agree to our terms found <a href='#'>here</a></p>
+
+  <div class='clearfix'>
+      <button type='button' class='cancelbtn'>Cancel</button>
+      <button type='submit' class='signupbtn'>Sign Up</button>
+    </div>
+  </div>
+
+</form>
```
Comment (Myuze) [inline views/register.handlebars:34]:
```suggestion
</form>

```

File: public/css/main.css (hunk 1 above)
Comment (LordLowdar) [reply public/css/main.css]: fixed

File: views/layouts/main.handlebars (hunk 3 above)
Comment (LordLowdar) [reply views/layouts/main.handlebars]: they arent, they are gone now.

**Comment by LordLowdar** [comment]
I also fixed some phrasing for the login itself.

**Review by mrogers23458 — APPROVED** [review]
looks good. cant wait to see it in action

**Review by Myuze — APPROVED** [review]
After you address the conflicts make sure to run 

- [ ] `npm run lint-fix`

**Comment by LordLowdar** [comment]
I need to run the npm I forgot
